Library/ViewModels/PledgeCTAContainerViewViewModelTests.swift

```diff
@@ -276,8 +276,8 @@ internal final class PledgeCTAContainerViewViewModelTests: TestCase {
 
     self.notifyDelegateCTATapped.assertValueCount(1)
 
-    XCTAssertEqual(["Project Page Pledge Button Clicked"], self.dataLakeTrackingClient.events)
-    XCTAssertEqual(["Project Page Pledge Button Clicked"], self.segmentTrackingClient.events)
+    XCTAssertEqual(["CTA Clicked"], self.dataLakeTrackingClient.events)
+    XCTAssertEqual(["CTA Clicked"], self.segmentTrackingClient.events)
```

Comment:
Can you check the `context_cta` property in here as well since we are doing it for the `session_` and `optimizely_` ones as well?